manifests/k/KDE/kmymoney/5.0.6/KDE.kmymoney.yaml

```diff
@@ -1,4 +1,4 @@
-PackageIdentifier: kde.kmymoney
+PackageIdentifier: KDE.kmymoney
 Publisher: kde
 PackageName: kmymoney
```

Comment:
Was going to let this pass if the only difference was lower letters but it appears to be a different Publisher..
![image](https://user-images.githubusercontent.com/15158490/129444474-23fdc2b9-f57b-4896-9a04-01c5166c5d1a.png)

```suggestion
Publisher: Humanity
PackageName: KMyMoney
```